### PR TITLE
Streamline react-query version

### PR DESCRIPTION
**Pull request by ivy-lli**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -3150,53 +3153,29 @@
...
+    "@tanstack/query-devtools" "5.32.1"
 
-"@tanstack/react-query@^5.17.9":
-  version "5.28.4"
-  resolved "https://registry.npmjs.org/@tanstack/react-query/-/react-query-5.28.4.tgz"
-  integrity sha512-BErcoB/QQG6YwLSUKnaGxF+lSc270RH2w3kMBpG0i4YzDCsFs2pdxPX1WVknQvFk9bNgukMb158hc2Zb4SdwSA==
+"@tanstack/react-query@5.32.1":
+  version "5.32.1"
+  resolved "https://registry.yarnpkg.com/@tanstack/react-query/-/react-query-5.32.1.tgz#ddb2c5aef342694ad288fac61e12269775c46161"
+  integrity sha512-+nXLMB0JK0XwTJ+lQt49DPNLrbSppni9N5W5yMR085yW3YaRKRUFhfVTER3TvQd1UycHpoGPFnt1gHiijXERAg==
```
Comment (ivy-lli) [inline yarn.lock:3176]: All editors now use the same tanstack/react-query version.
We should think about to fix this by not hoist the webview dependencies to the root node_modules...

File: extension/webviews/config-editor/src/index.tsx [hunk 2]
```diff
@@ -1,23 +1,31 @@
...
+  });
+  const queryClient = initQueryClient();
   createRoot(document.getElementById('root')!).render(
     <React.StrictMode>
-      <VariableEditor content={text} onChange={text => messenger.sendNotification(UpdateDocumentNotification, HOST_EXTENSION, { text })} />
+      <ClientContextProvider client={client}>
+        <QueryProvider client={queryClient}>
+          <VariableEditor app={''} pmv={''} file='/variables.yaml' />
+        </QueryProvider>
+      </ClientContextProvider>
```
Comment (ivy-lli) [inline extension/webviews/config-editor/src/index.tsx:28]: Merge the changes to the VariableEditor (so it uses a websocket and tanstack/react-query), broke the integration here.
Pls keep in mind that our npm packages always tries first to update to the latest npm releases.

File: extension/src/editors/config-editor/webview-communication.ts [hunk 3]
```diff
@@ -4,28 +4,46 @@ import { DisposableCollection } from '@eclipse-glsp/vscode-integration';
...
+      const result = { jsonrpc: message.jsonrpc, id: message.id, result: { context: message.params, data: document.getText() } };
+      messenger.sendNotification(ConfigWebSocketMessage, messageParticipant, JSON.stringify(result));
+    }
+    if (isSaveData(message)) {
+      const edit = new vscode.WorkspaceEdit();
+      edit.replace(document.uri, new vscode.Range(0, 0, document.lineCount, 0), message.params.data);
+      vscode.workspace.applyEdit(edit);
+      const result = { jsonrpc: message.jsonrpc, id: message.id, result: {} };
+      messenger.sendNotification(ConfigWebSocketMessage, messageParticipant, JSON.stringify(result));
+    }
```
Comment (ivy-lli) [inline extension/src/editors/config-editor/webview-communication.ts:36]: currently the vscode extension doesn't use the ivy-config-lsp from the backend, and writes directly reads / writes the file. We can always change this to something else later, e.g. more like the form editor works...

**Comment by ivy-lli** [comment]
@ivy-rew @ivy-edp fyi

File: extension/src/editors/config-editor/webview-communication.ts (hunk 3 above)
Comment (ivy-cst) [reply extension/src/editors/config-editor/webview-communication.ts:36]: I am not sure which way I prefer :question:

**Review by ivy-cst — APPROVED** [review]
Nice :+1:

File: extension/src/editors/config-editor/webview-communication.ts (hunk 3 above)
Comment (ivy-lli) [reply extension/src/editors/config-editor/webview-communication.ts:36]: I think we need to adjust this anyway as soon as we deliver meta data for this editor...
But at least it works again :)

File: extension/webviews/config-editor/src/index.tsx (hunk 2 above)
Comment (ivy-lgi) [reply extension/webviews/config-editor/src/index.tsx:28]: Yes, just recently changed the `VariableEditor` to work like this for the integration into the Eclipse Designer. Didn't get around to adjust it here until now. Thank you!

**Review by ivy-lgi — COMMENTED** [review]
LGTM

**Pull request merged**